code/game/machinery/computer/law.dm

```diff
@@ -6,10 +8,22 @@
 	circuit = /obj/item/circuitboard/aiupload
 	var/mob/living/silicon/ai/current = null
 	var/opened = FALSE
-
 	light_color = LIGHT_COLOR_WHITE
 	light_range_on = 2
+	var/cooldown = 0
+	var/foundlaws = 0
```

Comment:
Would be nice if you could add some autodoc comments to these to make it clear what they're doing.

Would be done like
```
/// here's a comment
var/cooldown = 0
```